specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json

```diff
@@ -381,6 +381,48 @@
       }
     }
   },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{serverName}/listGatewayStatus": {
```

Comment:
By design, the spec can be found at section 3.9 of http://aka.ms/asarm/
Get /servers/{servername} only contains the gateway configuration info.